src/api/Gemfile

```diff
@@ -143,8 +143,8 @@ group :test do
   # to ensure a clean state for testing
   gem 'database_cleaner-active_record'
   # for test coverage reports
-  gem 'codecov', require: false
   gem 'simplecov', require: false
```

Comment:
```suggestion
```
`simplecov` will become a dependency of `simplecov-cobertura`. Do we still need to include it explicitly?